cirq-core/cirq/circuits/frozen_circuit.py

```diff
@@ -88,11 +88,6 @@ def device(self) -> devices.Device:
     def __hash__(self):
         return hash((self.moments, self.device))
 
-    def diagram_name(self):
```

Comment:
Do you think it is safe to remove this?

Reply:
Seems fine to me - I only added it to handle the `CircuitOperation` tests.